cmd/lingo/main.go

```diff
@@ -103,10 +105,15 @@ func run() error {
 		return fmt.Errorf("starting manager: %w", err)
 	}
 
+	if err := mgr.AddReadyzCheck("readyz", healthz.Ping); err != nil {
```

Comment:
This is the always  `200` handler

Reply:
Does this start serving after `mgr.Start()`? If so, I think we need to move `.Bootstrap()` above `mgr.Start()`.

Reply:
This is a bit of a chicken/ egg problem. The `mgr.Start()` call also starts the cache syncs that we need for the `bootstrap` call. So with this implementation, the container may be "ready" but not `alive`, yet.
This can be solved with a custom handler that checks bootstrap was called once.
Good 👁️